comment/managers/followers.py

```diff
@@ -0,0 +1,47 @@
+from django.contrib.contenttypes.models import ContentType
+from django.db import models
+
+from comment.utils import get_username_for_comment
+
+
+class FollowerManager(models.Manager):
+    def is_following(self, email, model_object):
+        content_type = ContentType.objects.get_for_model(model_object)
+        return self.filter(email=email, object_id=model_object.id, content_type=content_type).exists()
+
+    def follow(self, email, username, model_object):
+        if not email or self.is_following(email, model_object):
+            return None
```

Comment:
You sometimes write `return` and sometimes `return None`. I understand both do the same work but I'm just curious if there's a reason or just the time of the day :smile:

Reply:
Nice question. In fact I use `return None` when the method explicitly returns a value and just `return` for void methods. I know both of them qualify to `None` but I feel it better for readability to use it this way. So whe I see `return` I know that this method is void in general.